### DISCUSSION TITLE
How are we bringing in content for the blog and project section?

### DISCUSSION THREAD

**Discussion by Evavic44** [chryz-hub/chryz-hub.github.io#179] [Ideas]
Recently the admins talked about providing content for the projects section and agreed to use our Repos as our projects. 

> #### Which means:  The project box would be attached to a link to the project Repo on GitHub 🆒 

#### Question ❓ 
- [x] I would like us to discuss what the links to the blog section would be.?
- [x] How contents would be served to the blog section; since frequent updates would be required for the blog?

**Comment by chryzcode** [comment]
I had in mind to create a blog site for ChryzHub. I've already had a way on how we can do this.
Which I and @namangupta1399 discussed yesterday. 

I also concluded I was going to share it out this Saturday during the admin meeting and concluding on it before we let the whole member our stand.

**Reply by namangupta1399** [reply]
Since it would probably take some time launch the blog site. I think, for the time being, we should use the existing platforms for blog. 
Later when our blog site is up and ready, we can move to our own and use it for future blogs.
For now, we can add links to the blogs that we will be adding to other platforms.

**Reply by chryzcode** [reply]
Yeah, truly said

**Reply by Evavic44** [reply]
> Since it would probably take some time launch the blog site. I think, for the time being, we should use the existing platforms for blog.
> Later when our blog site is up and ready, we can move to our own and use it for future blogs.
> For now, we can add links to the blogs that we will be adding to other platforms.

I agree 👍🏾

**Reply by namangupta1399** [reply]
Great ! Let's finalize this in next meeting, so that we can move ahead with website dev 🚧 🧰 🔨
